media/js/firefox/australis/browser-tour.js

```diff
@@ -1120,7 +1179,7 @@ if (typeof Mozilla == 'undefined') {
             } else {
                 localStorage.setItem(this.options.id, 'taken');
             }
-            gaTrack(['_setCustomVar', 5, 'First Time Taking Firefox Tour', firstTime, 2]);
+            return firstTime;
         } catch (e) {}
```

Comment:
I'd like to test this locally. Can we please rebase the branch against current master? It seems like there are quite a few tour related files here no longer in the repo since #2947 got merged.